### ISSUE TITLE
Invalid variable syntax: "name_prefix".

### ISSUE THREAD

**Issue by pditommaso** [kubenow/KubeNow#91]
Just tried to deploy a k8s cluster on AWS following the [tutorial](http://kubenow.readthedocs.io/en/latest/getting_started/bootstrap.html#bootstrap-on-amazon-web-services-ec2). However when I run the command: 

```
terraform apply aws
```

I get the following error message: 

```
There are warnings and/or errors related to your configuration. Please
fix these before continuing.

Errors:

  * invalid variable syntax: "name_prefix". If this is part of inline `template` parameter
then you must escape the interpolation with two dollar signs. For
example: ${a} becomes $${a}.
```

**Comment by mcapuccini** [comment]
Hi @pditommaso, thanks for opening the issue. What version of Terraform are you using?

**Comment by pditommaso** [comment]
Hi there. It's `0.8.7`.

**Comment by mcapuccini** [comment]
I see. We run the integration test using terraform 0.7.8, but we'll look into 0.8.x and fix this issue. In the meanwhile, would you ming trying with 0.7.8?

On the roadmap there is a *kubenow cli* which will wrap around a docker image with the tested versions of the provisioning tools. This should avoid such problems in the future.

**Comment by pditommaso** [comment]
Nice idea the Docker wrapper. With the old version terraform it run like a charm. Great work.

**Comment by mcapuccini** [comment]
It is now fixed for terraform 0.8
